src/crimson/osd/pg.cc

```diff
@@ -905,10 +905,12 @@ PG::interruptible_future<
              PG::interruptible_future<>>>
 PG::submit_transaction(
   ObjectContextRef&& obc,
+  ObjectContextRef&& clone_obc,
   ceph::os::Transaction&& txn,
   osd_op_params_t&& osd_op_p,
   std::vector<pg_log_entry_t>&& log_entries)
 {
+  auto _clone_obc = std::move(clone_obc);
```

Comment:
Why is it needed? Can't we just move clone_obc into submit_transaction later on?

Reply:
Ah, yes.